src/renderers/shaders/ShaderChunk/lights_template.glsl

```diff
@@ -98,6 +98,11 @@ IncidentLight directLight;
 
 	vec3 irradiance = getAmbientLightIrradiance( ambientLightColor );
 
+	#ifdef USE_SPHERICAL_HARMONICS
+	vec3 shIrradiance = shGetIrradianceAt( sphericalHarmonics, worldNormal );
+	irradiance += LinearToGamma(vec4(shIrradiance,1), float(GAMMA_FACTOR) ).xyz;
```

Comment:
Are you sure this is correct?  Our renderer is in linear space.

How do you know the spherical harmonics coefficient are in linear space?  I would think it is much more likely you have to go in the opposite direction: GammaToLinear.  But it depends on how the coefficients were created.

Reply:
@WestLangley this could be part of the issue with intensities not aligning.

Reply:
There also appears to be a left handed vs right-handed coordinate system issue. The irradiance computed in the PR appears to be mirrored relative to the env map.

Reply:
If there is a different in handedness I would suggest strandardizing on unity's approach?  I am not sure what is the dominant, canonical SH implementation, but we should try to identify that one and copy its standards rather than picking a random LH/RH for ours.  Basically I am taking this to mean which coefficient corresponds to which direction.

Reply:
@bhouston There is no handedness problem with my implementation.

Reply:
I'm not entirely sure wheter the coefficients I put in are gamma/linear, same about the handed-ness. I just gathered them from the Unity Editor and presumed (looking at Unity's shader + predefines). I'll look into it by creating a more meaningfull test cube as soon as I find time.

Reply:
Generally nice changes.  Although I am sure that linearToGamma is incorrect.  If the original source material was linear, then no conversion is needed, if the source material is gamma, you need GammerToLinear.  Our renderer is linear in this part, thus you need to either keep it in linear or convert gamma to linear.  There is no way that LinearToGamma is correct here.